README.md

```diff
@@ -3,3 +3,6 @@ Test
 Test
 
 TEST
+
+Am Montag vormittag haben wir ein Kennenlernen gehabt
```

Comment:
I would add: "Anja was my favorite member of the group" 🥰

Reply:
Test 123